ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -232,6 +247,8 @@ const genotypeDetails = (genotype, variant, genesById) => {
   ]
 }
 
+const addWarning = (warning, moreMessage) => (warning ? `${warning}.\n ${moreMessage}` : moreMessage)
```

Comment:
as above, making this into a string so you can split it later is confusing. Just keep it as an array